process/sync/metablock.go

```diff
@@ -376,6 +377,20 @@ func (boot *MetaBootstrap) doJobOnSyncBlockFail(hdr *block.MetaBlock, err error)
 			log.Info(errNotCritical.Error())
 		}
 	}
+
```

Comment:
I can see there is a lot of duplicated code between these two files, if there is common functionality, we should extract into a common interface and not have the code duplicated.

For now the hotfix can go in like this, but please open a refactor issue for this.

Reply:
Ok. Added task EN-3195 in jira for this